tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_call_log_groups.py

```diff
@@ -15,6 +15,7 @@ class TestCallLogGroups(GaiaTestCase):
     def setUp(self):
 
         GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('time.timezone', 'Europe/Istanbul')
```

Comment:
You mean, make use of the timezone similar as in London, UK? Then I would get the error as mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1210466#c4 .